### PR TITLE
feat!: CodeField replaces JsonField

### PR DESCRIPTION
**Pull request by benedictong-localz**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -123,10 +125,13 @@ export interface TextFieldSchema extends FieldSchema, FormController {
...
-export interface JsonFieldSchema extends FieldSchema, FormController {
-  type: 'json';
-  stringify?: boolean;
-  isCollapsable?: boolean;
+
+export type CodeLanguage = 'json' | 'html';
+
+export interface CodeFieldSchema extends FieldSchema, FormController {
+  type: 'code';
+  language: CodeLanguage;
```
Comment (tomanagle) [inline src/types.ts:133]: Does Monaco export an enum to use here?

Comment (benedictong-localz) [reply src/types.ts:133]: couldn't find it no :(

File: src/components/CodeField.tsx [hunk 2]
```diff
@@ -121,37 +150,22 @@ export const JsonField: FC<FieldProps<JsonFieldSchema>> = ({
...
-                        : value.jsObject
-                    );
+                  defaultLanguage={language}
+                  options={{
+                    minimap: { enabled: false },
+                    readOnly: isReadOnly || disabled || readOnly,
+                    autoClosingBrackets: 'always',
+                    autoClosingOvertype: 'always',
+                    automaticLayout: true,
+                    tabCompletion: 'on',
```
Comment (tomanagle) [inline src/components/CodeField.tsx:165]: I think we should bump the font-size here: `fontSize: 16` will match the other inputs

Comment (benedictong-localz) [reply src/components/CodeField.tsx:165]: 👍 
sure thing

**Review by tomanagle — APPROVED** [review]

**Comment by tomanagle** [comment]
I think the border needs to match too. I had a little play but couldn't figure out how to do it

**Pull request merged**